AsyncDisplayKit/Private/ASDisplayNode+AsyncDisplay.mm

```diff
@@ -131,8 +131,12 @@ - (void)_recursivelyRasterizeSelfAndSublayersWithIsCancelledBlock:(asdisplaynode
       CGContextTranslateCTM(context, frame.origin.x, frame.origin.y);
 
       //support cornerRadius
-      if (rasterizingFromAscendent && cornerRadius && clipsToBounds) {
```

Comment:
Curious what rasterizingFromAscendent was here for, if we know?  It's definitely not needed?